ops/cmd/import_devnet/main.go

```diff
@@ -72,36 +73,42 @@ func action(cliCtx *cli.Context) error {
 		return fmt.Errorf("failed to read manifest file: %w", err)
 	}
 
-	if len(m.L2.Chains) != len(st.AppliedIntent.Chains) {
-		return fmt.Errorf(
-			"number of chains in manifest file (%d) does not match number of chains in state file (%d)",
-			len(m.L2.Chains), len(st.AppliedIntent.Chains))
+	st, err := deployer.ReadOpaqueStateFile(statePath)
+	if err != nil {
+		return fmt.Errorf("failed to read opaque state file: %w", err)
+	}
+
+	numChains, err := st.GetNumChains()
+	if err != nil {
+		return fmt.Errorf("failed to read number of chains: %w", err)
+	}
+	if numChains != len(m.L2.Chains) {
```

Comment:
there should only ever be one chain in the manifest file or state file - can we chack for that here?

Reply:
In the case of interop devnets, there are always at least two chains in the state file (e.g. [interop-rc-alpha](https://github.com/ethereum-optimism/devnets/blob/main/alphanets/interop-rc-alpha/op-deployer/state.json)). We know the incoming devnet we want to add to the registry will have interop enabled, so we're trying to optimize for that known use-case

Reply:
ah good point